Automattic/wp-calypso
# E2E: add primary site attribute for jetpackUser in the secrets file.

#### Changes proposed in this Pull Request

This PR adds the `primarySite` attribute to the secrets file to fix https://github.com/Automattic/wp-calypso/issues/63988.

The test is failing due to the lack of `primarySite` attribute for the user `jetpackUser`. 
This attribute is required for the `DataHelper.getAccountSiteURL` to function.

#### Testing instructions

1. pull the branch.
2. run `yarn decrypt-secrets`.
3. view the `packages/calypso-e2e/src/secrets/decrypted-secrets.json` file and ensure the `primarySite` attribute exists for `jetpackUser`.

Ensure the following:
  - [x] Jetpack E2E (desktop)
  - [x] Jetpack E2E (mobile)

Related to https://github.com/Automattic/wp-calypso/issues/63988.
